jerry-core/debugger/debugger.c

```diff
@@ -160,7 +160,7 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
   }
 
   ecma_value_t message = result;
-  uint8_t type = JERRY_DEBUGGER_EVAL_RESULT;
+  uint8_t type = JERRY_DEBUGGER_EVAL_OK;
```

Comment:
Disregard this comment. We use the same identifier for a different purpose, so there is no source line change below.